GameServerLib/Logic/Game.cs

```diff
@@ -31,8 +32,8 @@ public class Game
         protected const int PEER_MTU = 996;
         protected const PacketFlags RELIABLE = PacketFlags.Reliable;
         protected const PacketFlags UNRELIABLE = PacketFlags.None;
-        protected const double REFRESH_RATE = 16.666; // 60 fps
-        private long _timeElapsed;
+        protected const double REFRESH_RATE = 1000.0 / 120.0; // 120 fps
```

Comment:
Default to 60. If the vanilla server tickrate really is 30, we should actually default to that.

If you want to run it at 120, make it configurable in a latter commit. Officially we should support whatever the vanilla is (so test environments and such whould be running with it, thus default it).